src/ar/noonscan/src/eu/kanade/tachiyomi/extension/ar/noonscan/NoonScan.kt

```diff
@@ -0,0 +1,37 @@
+package eu.kanade.tachiyomi.extension.ar.noonscan
+
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class NoonScan : MangaThemesia(
+    "نون سكان",
```

Comment:
Not sure if this should be just:
```suggestion
    "NoonScan",
```

Reply:
arabic is fine